### PR TITLE
[DX-202] Execute all the tests

### PR DESCRIPTION
**Pull request by matteofigus**
Fixes a bug introduced on #659 - the promise was resolved on the first file, not making all the tests to be correctly executed.

### CODE REVIEW

**Comment by matteofigus** [comment]
So, there are many other bugs. I think this PR attempts to fix the testing part and the build part (build needs to happen before the tests).

The Changelog generation connected to the version tasks seems not to work and I'll follow up with a fix in another PR

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,5 @@ src/components/base-component-jade/_package
 
 test/fixtures/test.tar.gz
 test/fixtures/targz-test
+
+logintervals.md
```
Comment (mattiaerre) [inline .gitignore]: add `newline` please,

File: package.json [hunk 2]
```diff
@@ -7,16 +7,17 @@
...
-    "build": "node tasks/build.js && npm run test",
-    "changelog": "node tasks/changelog.js",
+    "build": "node tasks/build.js",
     "git-stage": "node tasks/git-stage.js",
-    "major": "node tasks/version.js --type=\"major\"",
-    "minor": "node tasks/version.js --type=\"minor\"",
-    "patch": "node tasks/version.js --type=\"patch\"",
+    "major": "npm run version-major && npm run test-silent && npm run git-stage",
+    "minor": "npm run version-minor && npm run test-silent && npm run git-stage",
+    "patch": "npm run version-patch && npm run test-silent && npm run git-stage",
```
Comment (mattiaerre) [inline package.json]: you can also use the native `npm version <update_type>` to update version on npm btw

Comment (matteofigus) [reply package.json]: This is a bit different as it updates the component's version too.

File: .gitignore (hunk 1 above)
Comment (matteofigus) [reply .gitignore]: done

**Pull request merged**